dts/bindings/sensor/invensense,icm42670.yaml

```diff
@@ -14,6 +15,12 @@ properties:
       property value should ensure the flags properly describe the
       signal that is presented to the driver.
 
+  is-variant-s:
+    type: boolean
+    description: |
+      Specifies that the driver should adapt to the ICM-42670-S.
+      This affects the chip ID.
```

Comment:
Use different compatibles instead

Reply:
Thanks for this suggestion, I created different compatible.